src/createLoadableVisibilityComponent.js

```diff
@@ -22,63 +22,45 @@ function createLoadableVisibilityComponent (args, {
   LoadingComponent,
 }) {
   let preloaded = false
-  const visibilityHandlers = []
```

Comment:
Why did you remove this? It's to handle the case where `preload` is called after the component is part of the tree. In that scenario, we want all of the components to hit the underlying `LoadableComponent` to handle the interaction accordingly.

I believe by removing this and if `preload` is called after the component is mounted, then there will be a bug as it will fall back to the `IntersectionObserver` instead of already being loaded.

Of course, tests for this behaviour would be helpful 😅

Reply:
You're right, I think I was confused by the fact that the `this.state.visible` is set based on the `preloaded` var in the constructor, so I assumed that this was unnecessary.
 
I added the `visibilityHandlers` back in and [wrote a test](https://github.com/stratiformltd/react-loadable-visibility/pull/11/files#diff-f983afab730f1c0b922a76dceb0e5ca0R66) that demonstrates that the underlying component will be displayed when `preload` is called after mount.